examples/hpc-enterprise-slurm.yaml

```diff
@@ -255,10 +255,41 @@ deployment_groups:
       # https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/develop/community/modules/compute/schedmd-slurm-gcp-v5-partition#compute-vm-zone-policies
       zones: $(vars.gpu_zones)
 
+  - id: h3_node_group
+    source: community/modules/compute/schedmd-slurm-gcp-v5-node-group
+    settings:
+      node_count_dynamic_max: 16
+      machine_type: h3-standard-88
```

Comment:
I would still explicitly add "bandwidth_tier: gvnic_enabled" and the link to where it talks about the network characteristics.